.dockerignore

```diff
@@ -1,5 +1,31 @@
-Dockerfile
+.commitlintrc.json
+.cspell.json
 .dockerignore
+.editorconfig
+.env
+.env.sample
+.eslintrc.json
+.git
+.gitattributes
+.github
+.gitignore
+.husky
+.lintstagedrc.json
+.npmrc
+.prettierignore
+.prettierrc.json
+.whitesource
+nest-cli.json
+CHANGELOG.md
+Dockerfile
+LICENSE
+README.md
+dist
+docker-compose.db.yml
+docker-compose.yml
+jest.config.ts
 node_modules
+renovate.json
+tsconfig.build.json
```

Comment:
I don't know if this matters much, but this is the one I thought the docker file would use (and I thought it'd ignore `tsconfig.json`, because that one includes the `test` folder)

Reply:
I tried it, `npm run build` only needs `tsconfig.json` `tsconfig.build.json` in addition only specified which directories and files to ignore which I have specified in `.dockerignore` so it is safe to ignore and `docker-compose up --build` gets executed successfully.

Reply:
That makes sense: we would need to tell `nest build` that it should be reading `tsconfig.build.json` instead since `tsconfig.json` is the default. In that case we can delete `tsconfig.build.json` completely, nothing is using it from what I can see

Reply:
Agree, have deleted the file and removed from `.gitignore`.